plugins/juju/juju.plugin.zsh

```diff
@@ -0,0 +1,134 @@
+# ---------------------------------------------------------- #
+# Aliases and functions for juju (https://juju.is)           #
+# ---------------------------------------------------------- #
+
+# Load TAB completions
+_load_tab_completions() {
+  autoload -U +X compinit && compinit
+  autoload -U +X bashcompinit && bashcompinit
+  local jcomp="$(pkg-config --variable=completionsdir bash-completion)/juju"
```

Comment:
Is `pkg-config` assumed to always be available in all juju installations?

Reply:
It is preinstalled on [ubuntu](https://releases.ubuntu.com/20.04/ubuntu-20.04.3-desktop-amd64.manifest), not sure about others. I now cloned logic + comment from [gitfast plugin](https://github.com/ohmyzsh/ohmyzsh/blob/735808f48d54aabce540f6c90294e21118104cf4/plugins/gitfast/_git#L32). WDYT?